### PR TITLE
Additional has/owns verbs and risk:vulnerable

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/orgs.py [hunk 1]
```diff
@@ -223,6 +223,10 @@ def getModelDefs(self):
                     'doc': 'The contribution includes the specific node.'}),
                 ((None, 'meets', 'ou:requirement'), {
                     'doc': 'The requirement is met by the source node.'}),
+                (('ou:org', 'has', None), {
+                    'doc': 'The organization is or was in posession of target node.'}),
+                (('ou:org', 'owns', None), {
```
Comment (infime) [inline synapse/models/orgs.py:228]: maybe worth adding for the other applicable entities? I'm thinking some or all of ou:suborg, ou:team, ou:position, ou:user?

File: synapse/models/risk.py [hunk 2]
```diff
@@ -60,8 +60,12 @@ def getModelDefs(self):
                     'doc': 'A contiguous range of software versions which contain a vulnerability.'}),
 
                 ('risk:hasvuln', ('guid', {}), {
-                    'doc': 'An instance of a vulnerability present in a target.',
-                }),
+                    'deprecated': True,
+                    'doc': 'Deprecated. Please use risk:vulnerable.'}),
```
Comment (MichaelSquires) [inline synapse/models/risk.py:64]: Does this need a migration?

**Review by MichaelSquires — APPROVED** [review]

File: synapse/models/orgs.py [hunk 3]
```diff
@@ -223,6 +223,10 @@ def getModelDefs(self):
                     'doc': 'The contribution includes the specific node.'}),
                 ((None, 'meets', 'ou:requirement'), {
                     'doc': 'The requirement is met by the source node.'}),
+                (('ou:org', 'has', None), {
+                    'doc': 'The organization is or was in posession of target node.'}),
```
Comment (Cisphyx) [inline synapse/models/orgs.py:227]:
```suggestion
                    'doc': 'The organization is or was in possession of target node.'}),
```

File: synapse/models/person.py [hunk 4]
```diff
@@ -68,6 +68,16 @@ def getModelDefs(self):
                     'doc': 'The assessment that a given contact possesses a specific skill.'
                 }),
             ),
+            'edges': (
+                (('ps:contact', 'has', None), {
+                    'doc': 'The contact is or was in posession of target node.'}),
```
Comment (Cisphyx) [inline synapse/models/person.py:73]:
```suggestion
                    'doc': 'The contact is or was in possession of the target node.'}),
```

File: synapse/models/person.py [hunk 5]
```diff
@@ -68,6 +68,16 @@ def getModelDefs(self):
...
+                (('ps:person', 'has', None), {
+                    'doc': 'The person is or was in posession of target node.'}),
```
Comment (Cisphyx) [inline synapse/models/person.py:75]:
```suggestion
                    'doc': 'The person is or was in possession of the target node.'}),
```

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**